src/components/Amplitude.test.tsx

```diff
@@ -64,23 +70,68 @@ test("legacy", () => {
     </AmplitudeProvider>,
   );
 
-  expect(wrapper.find("#foo")).toHaveLength(1);
-  wrapper.find("#foo").simulate("click");
-  expect(wrapper.find("#bar")).toHaveLength(1);
-  wrapper.find("#bar").simulate("click");
+  expect(screen.getByTestId('foo')).toBeInTheDocument();
```

Comment:
I know that i'm the original culprit here, but I'm smarter, could this shift over to be getByRole() for some of this testing and remove the `id` style checks.